internal/tiltfile/tiltfile.go

```diff
@@ -114,11 +114,32 @@ func (tiltfile Tiltfile) GetServiceConfig(serviceName string) (*proto.Service, e
 		return nil, fmt.Errorf("error running '%v': %v", serviceName, err.Error())
 	}
 
-	service, ok := val.(k8sService)
-	if !ok {
-		return nil, fmt.Errorf("'%v' returned a '%v', but it needs to return a k8s_service", serviceName, val.Type())
+	switch service := val.(type) {
+	case compService:
+		var protoServ []*proto.Service
+
+		for _, cServ := range service.cService {
+			s, err := addToService(cServ)
+			if err != nil {
+				return nil, err
+			}
+
+			protoServ = append(protoServ, s)
+		}
+		return protoServ, nil
+	case k8sService:
+		s, err := addToService(service)
+		if err != nil {
+			return nil, err
+		}
+		return []*proto.Service{s}, nil
+
+	default:
+		return nil, fmt.Errorf("'%v' returned a '%v', but it needs to return a k8s_service or composite_service", serviceName, val.Type())
 	}
+}
 
+func addToService(service k8sService) (*proto.Service, error) {
```

Comment:
this isn't really adding anything any more - maybe rename to something like `skylarkServiceToProto`?